components/tree/Tree.tsx

```diff
@@ -11,6 +11,7 @@ import dropIndicatorRender from './utils/dropIndicator';
 import renderSwitcherIcon from './utils/iconUtil';
 
 export type SwitcherIcon = React.ReactNode | ((props: AntTreeNodeProps) => React.ReactNode);
+export type LeafIcon = React.ReactNode | ((props: AntTreeNodeProps) => React.ReactNode);
```

Comment:
```suggestion
export type TreeLeafIcon = React.ReactNode | ((props: AntTreeNodeProps) => React.ReactNode);
```

Reply:
Done. Let me know if you find anything else @afc163, otherwise this looks good to merge on my side.